### ISSUE TITLE
Vmagent vmagent.env.yaml 

### ISSUE THREAD

**Issue by ALEX-yinhao** [VictoriaMetrics/operator#221]
there I have a question:
I find that when vmagent start,it always use the prome config "/etc/vmagent/config_out/vmagent.env.yaml".
but I dont want to use it, I want create a configmap to use my custom config, and add vlumes like this,but it dosent help.
```
 volumeMounts:
    - name: vmagent-env
      mountPath: "/etc/vmagent/config_out"
  volumes:
    - name: vmagent-env
      configMap:
        name: vmagent-env
```
so if i want to use myself config, there is only way to use Additional Scrape Configuration ? the default prome config cant be modify ?

**Comment by f41gh7** [comment]
Hello, default mount point shouldn't be modified. AdditionalScrapeConfig designed for custom config and its better to use it. You can define some weird selectors for `VMAgent`, it must made scrape config empty. Something like:
```yaml

apiVersion: operator.victoriametrics.com/v1beta1
kind: VMAgent
metadata:
  name: example-vmagent
spec:
  additionalScrapeConfigs:
    name: additional-scrape-configs
    key: custom-config.yaml
  serviceScrapeSelector: 
   matchLabels:
     unexist: label
  podScrapeSelector: 
   matchLabels:
     unexist: label
  nodeScrapeSelector: 
   matchLabels:
     unexist: label
  probeSelector: 
   matchLabels:
     unexist: label
  staticScrapeSelector: 
   matchLabels:
     unexist: label
  replicaCount: 1
```

**Comment by f41gh7** [comment]
No, it will be mounted by operator. Just define reference for secret at ScrapeConfigs.

**Comment by ALEX-yinhao** [comment]
when i update the secret, the config of pod will be update but the pod wont be reload . why? and sometimes the secret will be deleted and i dont know how it hanppend.......

**Comment by f41gh7** [comment]
> when i update the secret, the config of pod wont be update and reload . why? and sometimes the secret will be deleted and i dont know how it hanppend.......

 Secret deletion and config update will be fixed at the next release.

 Also, it will include `spec.inlineScrapeConfig` with raw yaml configuration file.

**Comment by f41gh7** [comment]
Fixes included to the `0.14.0` release, feel free to reopen.
